WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -74,6 +77,9 @@ protected override async Task ActionAsync(CancellationToken cancel)
 
 			// Ensure there's at least one non-blame round in input registration.
 			await CreateRoundsAsync(cancel).ConfigureAwait(false);
+
+			// RoundStates have to contain all states. Do not change stateId=0.
+			RoundStates = Rounds.Select(r => RoundState.FromRound(r, stateId: 0)).ToImmutableArray();
```

Comment:
If the stateId is always 0 then it would worth to remove it from the parameter list.